RFC Draft.md

```diff
@@ -535,6 +535,111 @@ function subscribeToCreate() {
 }
 ```
 
+## Flatten the GraphQL operations' responses
+As alluded to in the prior suggestion, we're looking to flatten the results for the GraphQL operations to make them more easily accessible instead of the current three-levels-deep nested object. Would love to get your understanding on which option you prefer.
```

Comment:
```suggestion
As alluded to in the previous section, we're looking to flatten the results of GraphQL operations to make them more easily accessible instead of the current three-levels-deep nested object. Would love to get your understanding on which option you prefer.
```